integration-tests/src/main/resources/init_script.cql

```diff
@@ -0,0 +1,3 @@
+CREATE KEYSPACE IF NOT EXISTS k1 WITH replication = {'class':'SimpleStrategy', 'replication_factor':1};
+CREATE TABLE IF NOT EXISTS k1.test_NameConverterEntity(test_entityId int primary key);
```

Comment:
Not related to this PR but just because I stumble upon it: very weird table name (used mixed-case letters, not recommended) and very weird schema (only one partition key column).

Reply:
This name is done on purpose to exercise the `customConverterClass` and `TestNameConverter` that uses `@RegisterForReflection` with mapper

Reply:
ok, not a problem for this PR anyways.